src/MICore/CommandFactories/gdb.cs

```diff
@@ -183,6 +183,25 @@ public override async Task<List<ulong>> StartAddressesForLine(string file, uint
             return addresses;
         }
 
+        private async Task JumpInternal(string target)
+        {
+            // temporary breakpoint + jump
+            await _debugger.CmdAsync("-break-insert -t " + target, ResultClass.done);
+            await _debugger.CmdAsync("-exec-jump " + target, ResultClass.running);
```

Comment:
`-exec-jump` will resume all threads, we may need to call `set scheduler-locking on` to ensure that it only runs the current thread. Then revert back to `set scheduler-locking replay`.

Reply:
All stop mode shouldn't be assumed either.
https://sourceware.org/gdb/current/onlinedocs/gdb/Non_002dStop-Mode.html#Non_002dStop-Mode could be enabled, I'm currently looking into this as we have threads that need to keep running or the test will crash.

Reply:
There potentially can be a race where another thread hits a breakpoint before we hit this the temporary one created for `jump`.

Reply:
But is that an assumption or did you actually encounter that problem?
Cause the docs say "Resume execution at line linespec. Execution stops again immediately if there is a breakpoint there."

Reply:
This is a potential issue. I think we can mark this with a note if users ever run into this problem if GDB starts other threads when it resumes from the jump.

This should be fine if it does "immediately" stop.